src/distilbert/distilbert_model.rs

```diff
@@ -682,30 +682,59 @@ impl DistilBertForTokenClassification {
     /// let config = DistilBertConfig::from_file(config_path);
     /// let distil_bert = DistilBertForTokenClassification::new(&p.root() / "distilbert", &config);
     /// ```
-    pub fn new<'p, P>(p: P, config: &DistilBertConfig) -> DistilBertForTokenClassification
-    where
-        P: Borrow<nn::Path<'p>>,
+    pub fn try_new<'p, P>(p: P, config: &DistilBertConfig) -> Result<DistilBertForTokenClassification, RustBertError>
```

Comment:
The API is still undergoing regular re-designs and breaking changes will be pushed for the next release in a few weeks. I would be fine introducing a consistent `new` method for the `SequenceClassificationModels` that would be breaking and avoid adding a `try_new` method to try and keep the API uncluttered. What do you think?

Reply:
Great idea - done.